internal/cmd/flags.go

```diff
@@ -8,51 +8,6 @@ import (
 	"time"
 
 	"github.com/spf13/pflag"
-	"github.com/urfave/cli/v2"
```

Comment:
Why do we still use urfave/cli?

Reply:
That’s why we need this PR to clean up dead package code.